src/Apps/Artist/routes.tsx

```diff
@@ -47,10 +47,27 @@ export const routes = [
       {
         path: "/",
         Component: OverviewRoute,
+        prepareVariables: (params, props) => {
+          return { ...props.location.query, ...params }
```

Comment:
Have you tested if Relay does any validation on the variables? Should we? e.g. `for_sale=fsdj`

Reply:
(Definitely not a biggie, as I don’t expect people to construct these URLs by hand.)

Reply:
I checked this (but didn't dig into the code), but it looks like you are limited to the variables declared by the query (for instance `?blah=true` isnt passed in).

Something like `?medium=blah` _will_ get passed in, which I suppose is fine.

Reply:
Gotcha, that makes sense, but I was thinking more about `for_sale=hgads` not being a boolean. I guess it may get treated as a truth value which, just like your `medium=blah` example is, I suppose fine 👍